object/type_string.go

```diff
@@ -21,12 +21,13 @@ func _() {
 	_ = x[MAP-10]
 	_ = x[QUOTE-11]
 	_ = x[MACRO-12]
-	_ = x[LAST-13]
+	_ = x[EXTENSION-13]
+	_ = x[LAST-14]
 }
 
-const _Type_name = "UNKNOWNINTEGERFLOATBOOLEANNILERRORRETURNFUNCSTRINGARRAYMAPQUOTEMACROLAST"
+const _Type_name = "UNKNOWNINTEGERFLOATBOOLEANNILERRORRETURNFUNCSTRINGARRAYMAPQUOTEMACROEXTENSIONLAST"
 
-var _Type_index = [...]uint8{0, 7, 14, 19, 26, 29, 34, 40, 44, 50, 55, 58, 63, 68, 72}
+var _Type_index = [...]uint8{0, 7, 14, 19, 26, 29, 34, 40, 44, 50, 55, 58, 63, 68, 77, 81}
```

Comment:
This is cryptic

Can't you use something else?

I mean relying on a huge string and magical offset?

Please at least  add a comment about such "hack"

Reply:
read the "generated code, do not edit" at the top:)

this is done by go:generate and stringer, unfortunately it's checked in because go can't agree to add some go:generate at build/install time support

Reply:
OK … so cryptic …

Reply:
the code it generates is pretty cool... just that... I wish I didn't have to checkin generated code, I hate doing that